src/agent/em_agent.cpp

```diff
@@ -852,9 +887,33 @@ void em_agent_t::input_listener()
         return;
     }
 
+    for (int i = 0; i < MAX_NUM_RADIOS; i++) {
+        // OneWifi bus path is 1-indexed, here
+        std::string cce_ind_path = "Device.WiFi.Radio." + std::to_string(i + 1) + ".CCEInd";
+        em_printfout("Attempting to subscribe to '%s'", cce_ind_path.c_str());
+        if (desc->bus_event_subs_fn(&m_bus_hdl, cce_ind_path.c_str(), reinterpret_cast<void *>(&em_agent_t::cce_ie_cb), nullptr, 0) != 0) {
+            // Failing to subscribe to this path is OK, as in the context of DPP,
+            // an Enrollee has a default channel list defined by the EasyConnect spec,
+            // as well as a list of channels defined in its' DPP URI. So, just log and move on.
+            em_printfout("Failed to subscribe to '%s'", cce_ind_path.c_str());
```

Comment:
```suggestion
            em_printfout("Failed to subscribe to '%s', DPP Dynamic Channel List generation is not available.", cce_ind_path.c_str());
```
(In case someone is trying to figure out why that doesn't work from the logs)